internal/controller/common/pmm_configs.go

```diff
@@ -33,24 +39,27 @@ var (
 	// A pmmResourceRequirementsSmall is the resource requirements for PMM for small clusters.
 	pmmResourceRequirementsSmall = corev1.ResourceRequirements{
 		Requests: corev1.ResourceList{
-			corev1.ResourceMemory: resource.MustParse("97.27Mi"),
-			corev1.ResourceCPU:    resource.MustParse("95m"),
+			// 97.27Mi = 97 MiB + 276 KiB = 99604 KiB
+			corev1.ResourceMemory: *resource.NewQuantity(97*1024*1024+276*1024, resource.BinarySI),
+			corev1.ResourceCPU:    *resource.NewScaledQuantity(pmmClientRequestCPUSmall, resource.Milli),
```

Comment:
I'd suggest that we use consts for better clarity and readibility:

First declare them like this:
```go
const (
	kebibyte = 1024
	mebibyte = 1024 * kebibyte
)
```

Then use them here:
```suggestion
			// 97.27Mi = 97 MiB + 276 KiB = 99604 KiB
			corev1.ResourceMemory: *resource.NewQuantity(97*mebibyte+276*kebibyte, resource.BinarySI),
			corev1.ResourceCPU:    *resource.NewScaledQuantity(pmmClientRequestCPUSmall, resource.Milli),
```